### DISCUSSION TITLE
下書き時のプレビュー表示でも自動生成の目次を確認したい

### DISCUSSION THREAD

**Discussion by k1350** [increments/qiita-discussions#82] [要望・アイデア・不具合]
記事公開時にマークダウンの見出しレベルに応じた目次が自動生成されますが、下書き時のプレビュー表示でも目次を確認できると嬉しいです。
現在、見出しレベルの誤りが無いか確認するためだけに、全体公開前に限定共有状態で確認しています。

**Comment by kskwtnk** [comment]
@k1350 

ご意見ありがとうございます！

こちら、私も非常に欲しいと思っています……！笑

現在 https://github.com/increments/qiita-discussions/projects で簡単に共有しているのみですが、エディタの改善をしています
そちらの内容が完了したら更なるアップデートを実施していこうと思っていますので、そのタイミングで検討させていただければと思います！

**Comment by ko1nksm** [comment]
私からもお願いします。アウトラインがわからないと記事をうまく構成するのが難しいです。ただ目次 (TOC) 表示のために編集やプレビューの幅が狭くなりすぎると困るので、その点は工夫していただけると助かります。ポップアップ表示や表示非表示を切り替えらるようにするとかです。

**Comment by vanchan2625** [comment]
初めてQiitaで記事を書いたのですが、下書きで目次が出ずにめちゃくちゃ調べました。
結果このDiscussionに行きつき、理解することができ感謝しております。
一方で私のように初学者は同じように混乱しているのではないかと思います。

この機能が実装されてくれることがベストですが、「下書き時のプレビューに目次は表示されません」という文言がどこかに入っていれば親切なのかなとも思いました。
（もしどこかに記載があるなら申し訳ありません。私の確認不足です。）

**Reply by tanaka-asami415** [reply]
@vanchan2625 
ご不便をおかけして申し訳ありません。
ご意見もありがとうございます。参考にさせていただきます🙇

また、時期はお約束できないのですが、「下書き時（エディタページ）に目次機能を追加する」実装を具体的に進める予定です。
